airbyte-ci/connectors/pipelines/pipelines/consts.py

```diff
@@ -53,6 +56,8 @@
 PIP_CACHE_PATH = "/root/.cache/pip"
 POETRY_CACHE_VOLUME_NAME = "poetry_cache"
 POETRY_CACHE_PATH = "/root/.cache/pypoetry"
+STORAGE_DRIVER = "fuse-overlayfs"
```

Comment:
I'm not sure that this constant makes sense. It's only a coincidence that its alpine package name is the same string as the value in the dockerd config.

Reply:
This is actually something that augustin added, so I'm a bit loath to touch it, but maybe we can fix this in a followup pr cc @alafanechere